pkg/harvester/edit/loadbalancer.harvesterhci.io.ippool/Priority.vue

```diff
@@ -48,6 +48,22 @@ export default {
     showProjectAndCluster() {
       return !this.isStandaloneHarvester;
     },
+
+    uniqNamespaceOptions() {
+      const selectedNamespaces = this.rows.map(row => row.namespace);
+
+      const out = (this.allNamespaces || []).filter(n => !selectedNamespaces.includes(n.id)).map((namespace) => {
+        return {
+          label: namespace.metadata.name,
+          value: namespace.id,
+        };
+      });
+
+      return [{
+        label: this.t('generic.all'),
+        value: '*',
```

Comment:
<img width="1527" alt="image" src="https://github.com/harvester/dashboard/assets/24985926/2e7c9314-3e96-4d16-8152-1246e8a8780b">
<img width="1497" alt="image" src="https://github.com/harvester/dashboard/assets/24985926/1b7c845d-e515-4f45-b7e0-258f26b953d0">

"All" should encompass all scenarios, which means that when a user selects "All," the other options cannot be chosen, including selecting "All" twice.

Reply:
I agree with the de-duplication of `All`, but I have a different opinion on the `All` and `default` scenario

As a selector, the option "all" is mutually exclusive with other options, which makes it difficult to interact on UX.

For example, when I add the first selector, the default namespace value is `All`, is it not allowed to add namespace `default` in this scenario?